database/datagenerator.js

```diff
@@ -0,0 +1,59 @@
+const faker = require('faker');
+const data = require('fs').createWriteStream('database/files/data.csv');
+const images = require('fs').createWriteStream('database/files/images.csv');
+const tags = require('fs').createWriteStream('database/files/tags.csv');
+const types = require('fs').createWriteStream('database/files/types.csv');
+
+const generateData = async (callback) => {
+    
+    const cuisine = ['Japanese', 'Korean', 'Chinese', 'French', 'German', 'American', 'Mexican',
+                    'Vietnamese', 'Taiwanese', 'Brazilian', 'Italian', 'Martian', 'Connecticotian',
+                     'Indian', 'Jamaican', 'Seafood', 'Ethiopian', 'Mediterranean', 'Californian'];
+    // write headers for every csv file
+    data.write('id,name,description,reviews,priceStart,priceEnd\n');
+    images.write('id,listingId,imgUrl,caption\n');
+    tags.write('id,listingId,tag\n');
+    types.write('id,listingId,type\n');
```

Comment:
Personaly, I might put the headers section above the "cuisine" variable creation, but its your call